cscs-checks/apps/cp2k/cp2k_check.py

```diff
@@ -13,11 +13,15 @@ def __init__(self, check_name, check_descr, **kwargs):
         self.executable = 'cp2k.psmp'
         self.executable_opts = ['H2O-256.inp']
 
+        energy = sn.extractsingle(r'\s+ENERGY\| Total FORCE_EVAL \( QS \) '
+                                  r'energy \(a.u.\):\s+(?P<energy>\S+)',
```

Comment:
The dots should be escaped here, if you actually mean the `.` character.

Reply:
Nice catch!